.github/workflows/basic.yml

```diff
@@ -45,7 +48,7 @@ jobs:
           command: wasm
           args: --locked --workspace --exclude ampd
```

Comment:
ampd is being excluded here. should that be checked separately?

Reply:
CJ mentions it's because the command is for compiling wasm contracts, and ampd doesn't have any wasm contracts.